apps/studio/components/interfaces/Auth/AuthProvidersFormValidation.tsx

```diff
@@ -1053,6 +1053,44 @@ const EXTERNAL_PROVIDER_LINKEDIN_OIDC = {
   },
 }
 
+const EXTERNAL_PROVIDER_SLACK_OIDC = {
+  $schema: JSON_SCHEMA_VERSION,
+  type: 'object',
+  title: 'Slack (OIDC)',
+  properties: {
+    EXTERNAL_SLACK_OIDC_ENABLED: {
+      title: 'Slack enabled',
+      type: 'boolean',
+    },
+    EXTERNAL_SLACK_OIDC_CLIENT_ID: {
+      title: 'API Key',
+      type: 'string',
+    },
+    EXTERNAL_SLACK_OIDC_SECRET: {
+      title: 'API Secret Key',
+      type: 'string',
+      isSecret: true,
+    },
+  },
```

Comment:
Since when do we do this?

Reply:
Do you mean the validation schema? Not too sure - referencing what we did for [LinkedIn OIDC](https://github.com/supabase/supabase/pull/18225/files#diff-c842696488522c2e7915e506fc71296714db67c2f9c483a3dcf903d88c267a27)

There's more bits to update, including docs but wanted to get a minimal version out for testing.